app/js/containers/HomeContainer.jsx

```diff
@@ -1,13 +1,39 @@
 import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Row, Col, NavLink, Button } from 'reactstrap';
+import { Link } from "react-router-dom";
+import {withNamespaces} from 'react-i18next';
 
 class HomeContainer extends Component {
   render() {
+    const t = this.props.t;
     return (
       <Fragment>
-        <h1>Home</h1>
+        <Row>
+          <Col>
+            <NavLink className="float-right" tag={Link} to="/login">{t('home.login')}</NavLink>
+          </Col>
+        </Row>
+        <Row className="h-75">
+          <Col xs={12} className="my-auto">
+            <h1 className="text-center">{t('home.welcome')}</h1>
+          </Col>
+        </Row>
+        <Row className="mt-5">
+          <Col xs={6}>
+            <Button tag={Link} color="info" block to="/buy">{t('home.buy')}</Button>
+          </Col>
+          <Col xs={6}>
+            <Button tag={Link} color="info" block to="/sell">{t('home.sell')}</Button>
```

Comment:
Do these links bring somewhere right now?

Reply:
Does an empty page count as somewhere?

Other pages have to be built :)

Reply:
Haha, yeah that's what I was expecting. It's fine for now. We could have pointed it to myEscrows for sell and the map for buy but it's gonna change anyway so no big deal